standards/rmrk2.0.0/interactions/setattribute.md

```diff
@@ -4,72 +4,80 @@ Discussing here: https://app.clickup.com/6615036/docs/69vzw-1501/69vzw-121
 
 The `SETATTRIBUTE` interaction allows NFT owners to change certain mutable values on their NFTs.
 
-The types of on-chain attributes susceptible to this interaction are: mutable, conditional, logic.
+The types of on-chain properties susceptible to this interaction are: mutable, conditional, logic.
 
 ## Mutable
 
-Attributes are mutable if they are defined to be mutable by the minter (see
+Properties are mutable if they are defined to be mutable by the minter (see
 [NFT On-chain Attributes](../entities/nft.md#on-chain-attributes)).
 
-A mutable attribute is changed with a SETATTRIBUTE call:
+A mutable property is changed with a SETATTRIBUTE call:
 
 ```txt
 rmrk::SETATTRIBUTE::2.0.0::{id}::{html_encoded_name}::{html_encoded_value}
 ```
 
-Example. Given an NFT `5105000-0aff6865bed3a66b-DLEP-DL15-00000001` with an attribute (either
+Example. Given an NFT `5105000-0aff6865bed3a66b-DLEP-DL15-00000001` with a property (either
 inherited from Collection or directly defined on NFT):
 
 ```json
-{
-  "mutable": true,
-  "trait_type": "Hair color",
-  "value": "blue"
-}
+"properties":  {
+  "color": {
+    "_mutation":  {
+      "allowed": true
+    },
+    "type": "string",
+    "value": "blue"
+  }
+},
 ```
 
 We can change the color to red like so:
 
 ```txt
-rmrk::SETATTRIBUTE::2.0.0::5105000-0aff6865bed3a66b-DLEP-DL15-00000001::Hair%20color::red
-```
+RMRK::SETATTRIBUTE::2.0.0::5105000-0aff6865bed3a66b-DLEP-DL15-00000001::color::%7B%22type%22%3A%22string%22%2C%22value%22%3A%22red%22%7D

```

Comment:
Why use type and value here when it's implied that you can only change value and that type will match? Should be doable in the `color::red` way, no?

Reply:
what if you want to change a type too, so from int to string?

Reply:
I don't think that should be allowed

Reply:
Alright, will edit consolidator and change this part in spec